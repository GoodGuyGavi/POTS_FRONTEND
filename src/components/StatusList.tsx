import React from "react";

import styled from "styled-components";
import StatusItem from "./StatusItem";
import HeaderTitle from "../components/HeaderTitle";

export default class StatusList extends React.Component {
  render() {
    return (
      <div style={{ marginLeft: 20, marginTop: 10 }}>
        <HeaderTitle header="Today's Status" />
        <StatusItem
          OR="3"
          label="Orders Received"
          label2="Released from supplier"
          label3="On-hold from supplier"
        />
      </div>
    );
  }
}
