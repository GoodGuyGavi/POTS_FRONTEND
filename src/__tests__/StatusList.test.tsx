/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
// import ShallowRenderer from "react-test-renderer/shallow";

// // Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";

import StatusList from "../components/StatusList";

const props = {
  title: "Title"
};

describe(" Status List", () => {
  test("renders correctly", () => {
    expect(shallow(<StatusList {...props} />)).toMatchSnapshot();
  });
});
