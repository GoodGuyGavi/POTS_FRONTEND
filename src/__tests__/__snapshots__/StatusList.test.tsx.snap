// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Status List renders correctly 1`] = `ShallowWrapper {}`;
